### DISCUSSION TITLE
integration with "home-assistant.io"

### DISCUSSION THREAD

**Discussion by JosepFarreLlados** [mikejac/node-red-contrib-google-smarthome#431] [General]
Hello!
First of all recognize the very good work on this Node-RED nodes, works perfect!!
Nowadays I'm integrating [home-assistant.io ](https://www.home-assistant.io/) and I have a question rather than an issue. It is possible connect my nodes created with [node-red-contrib-google-smarthome](https://flows.nodered.org/node/node-red-contrib-google-smarthome) at my [home-assistant.io ](https://www.home-assistant.io/)?
Thanks in advance!!

**Comment by Caprico85** [comment]
Hi,

There is no automatic way to connect to Home Assistant. You would need node-red-contrib-home-assistant-websocket or somethin similar to connect to Home Assistant. And you would have to write a lot of function nodes to forward and convert messages between node-red-contrib-google-smarthome and Home Assistant.

However, I never really used Home Assistant though (just played around with it a while ago), so I'm not of much help on this.

Regards
